src/main/java/io/stargate/sgv2/jsonapi/service/operation/SchemaAttempt.java

```diff
@@ -38,6 +45,24 @@ protected Uni<AsyncResultSet> executeStatement(CommandQueryExecutor queryExecuto
 
   protected abstract SimpleStatement buildStatement();
 
+  // Add customProperties which has table properties for vectorize
```

Comment:
Moved it to schema attempt since it's needed for both Create table and Alter table commands